src/json_stream/tests/test_collection_abcs.py

```diff
@@ -0,0 +1,17 @@
+from typing import Mapping, Sequence
+
+from json_stream.select_tokenizer import default_tokenizer
+from json_stream.tests import JSONLoadTestCase
+
+
+class TestCollectionABCs(JSONLoadTestCase):
+
+    def test_isinstance_sequence(self):
+        for persistence in (True, False):
```

Comment:
you're mixing up persistent/persistence in your variable names, same in the other case

Reply:
Oh, that's embarrassing. I must have decided to change the variable name *after* successfully running the tests...